src/components/Grid/Grid.scss

```diff
@@ -1,12 +1,12 @@
 #grid {
-    display: grid;
-    grid-template-columns: repeat(10, 10%);
-    background-color: #beab95;
-    height: calc(100vh - 190px);
-    width: calc(100vh - 190px);
-    border-radius: 15px;
-    flex-shrink: 0;
-    justify-items: center;
-    align-items: center;
-}
-
+  display: grid;
+  grid-template-columns: repeat(10, 10%);
+  background-color: #beab95;
+  height: calc(100vh - 190px);
+  width: calc(100vh - 190px);
+  border-radius: 15px;
+  flex-shrink: 0;
+  justify-items: center;
+  align-items: center;
+  grid-auto-rows: minmax(0, 1fr);
```

Comment:
what's this line do?

Reply:
line 11 keeps the gap between rows consistent regardless of enabled/disabled cells. this fixed the cell shifting bug